ts/src/exmo.ts

```diff
@@ -1280,19 +1292,32 @@ export default class exmo extends Exchange {
          * @method
          * @name exmo#fetchMyTrades
          * @description fetch all trades made by the user
-         * @param {string} symbol unified market symbol
+         * @see https://documenter.getpostman.com/view/10287440/SzYXWKPi#b8d8d9af-4f46-46a1-939b-ad261d79f452  // spot
+         * @see https://documenter.getpostman.com/view/10287440/SzYXWKPi#f4b1aaf8-399f-403b-ab5e-4926d967a106  // margin
+         * @param {string} symbol a symbol is required but it can be a single string, or a non-empty array
          * @param {int} [since] the earliest time in ms to fetch trades for
-         * @param {int} [limit] the maximum number of trades structures to retrieve
+         * @param {int} [limit] *required for margin orders* the maximum number of trades structures to retrieve
          * @param {object} [params] extra parameters specific to the exmo api endpoint
+         *
+         * EXCHANGE SPECIFIC PARAMETERS
+         * @param {int} [params.offset] last deal offset, default = 0
          * @returns {Trade[]} a list of [trade structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#trade-structure}
          */
-        // a symbol is required but it can be a single string, or a non-empty array
         if (symbol === undefined) {
             throw new ArgumentsRequired (this.id + ' fetchMyTrades() requires a symbol argument (a single symbol or an array)');
         }
+        let marginMode = undefined;
+        [ marginMode, params ] = this.handleMarginModeAndParams ('fetchMyTrades', params);
+        if (marginMode === 'cross') {
+            throw new BadRequest (this.id + 'only isolated margin is supported');
+        }
         await this.loadMarkets ();
         let pair = undefined;
         let market = undefined;
+        const isSpot = marginMode !== 'isolated';
+        if (limit === undefined) {
+            limit = 100;
+        }
         if (Array.isArray (symbol)) {
```

Comment:
@samgermain accepting an array of symbols here is very unusual and I'm not sure we should do it or not, this goes against the typing we have

Reply:
```
% exmo fetchMyTrades BTC/USDT
2023-09-07T16:35:41.637Z
Node.js: v18.15.0
CCXT v4.0.83
exmo.fetchMyTrades (BTC/USDT)
2023-09-07T16:35:45.536Z iteration 0 passed in 516 ms

       id |     timestamp |                 datetime |   symbol |       order | type | side | takerOrMaker |   price | amount |     cost |                                          fee |                                           fees
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
440793834 | 1670881756000 | 2022-12-12T21:49:16.000Z | BTC/USDT | 32772462607 |      |  buy |        taker | 17196.1 | 0.0008 | 13.75688 | {"cost":8e-7,"currency":"BTC","rate":0.0001} | [{"cost":8e-7,"currency":"BTC","rate":0.0001}]
1 objects
2023-09-07T16:35:45.536Z iteration 1 passed in 516 ms
```